client/signup/config/steps-pure.js

```diff
@@ -349,6 +343,23 @@ export function generateSteps( {
 			dependencies: [ 'siteSlug', 'domainItem' ],
 			providesDependencies: [ 'cartItem' ],
 		},
+
+		'mailbox-plan': {
+			stepName: 'mailbox-plan',
+			apiRequestFunction: addPlanToCart,
+			fulfilledStepCallback: isPlanFulfilled,
+			dependencies: [ 'siteSlug', 'emailItem' ],
+			providesDependencies: [ 'cartItem' ],
+			props: {
+				get headerText() {
+					return i18n.hasTranslation( 'Empower your online presence' )
```

Comment:
We also need the locale check here, I was getting the default headers even though my locale is `en`

<img width="781" alt="Screen Shot 2022-09-06 at 2 24 42 PM" src="https://user-images.githubusercontent.com/104910361/188710657-295bfe0e-278d-4c23-920f-e8319e09f116.png">

<img width="414" alt="Screen Shot 2022-09-06 at 2 26 33 PM" src="https://user-images.githubusercontent.com/104910361/188711078-38dde28c-34c5-4066-8abf-3b5b465fee4b.png">